<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatanWindow</class>
 <widget class="QMainWindow" name="CatanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CatanWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="boardView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Resources Distributed: 0</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>791</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Points Over Time</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>141</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buildCampsiteButton">
       <property name="text">
        <string>Build Campsite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buildSpaceStationButton">
       <property name="text">
        <string>Build Space Station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endTurnButton">
       <property name="text">
        <string>End Turn</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="diceButton">
       <property name="text">
        <string>Roll Dice</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="diceOne">
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="midLineWidth">
          <number>12</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="diceTwo">
         <property name="font">
          <font>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>474</y>
      <width>581</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="p1Label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:red;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <property name="text">
          <string>Player 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2Label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:green;</string>
         </property>
         <property name="text">
          <string>Player 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3Label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:blue;</string>
         </property>
         <property name="text">
          <string>Player 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="p1_points">
         <property name="text">
          <string>Points: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2_points">
         <property name="text">
          <string>Points: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3_points">
         <property name="text">
          <string>Points: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="p1_metal">
         <property name="text">
          <string>Metal: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2_metal">
         <property name="text">
          <string>Metal: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3_metal">
         <property name="text">
          <string>Metal: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="p1_stone">
         <property name="text">
          <string>Stone: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2_stone">
         <property name="text">
          <string>Stone: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3_stone">
         <property name="text">
          <string>Stone: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="p1_oxygen">
         <property name="text">
          <string>Oxygen: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2_oxygen">
         <property name="text">
          <string>Oxygen: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3_oxygen">
         <property name="text">
          <string>Oxygen: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="p1_used">
         <property name="text">
          <string>Used: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p2_used">
         <property name="text">
          <string>Used: 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="p3_used">
         <property name="text">
          <string>Used: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="buildingView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>624</width>
      <height>474</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="instructionsLabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
